services/libs/aster-std/src/fs/utils/inode.rs

```diff
@@ -245,6 +246,14 @@ pub trait Inode: Any + Sync + Send {
 
     fn set_mode(&self, mode: InodeMode);
 
+    fn owner(&self) -> Uid;
+
+    fn set_owner(&self, uid: Uid);
+
+    fn group(&self) -> Gid;
+
+    fn set_group(&self, gid: Gid);
```

Comment:
Should this method be fallable?

Reply:
For some filesystems (e.g., NFS, FAT32), these methods are not supported and should return an error.